### ISSUE TITLE
we can't reproduce the results of your experiment,could you give us some suggestion or help,thanks!!!

### ISSUE THREAD

**Issue by zoesoon** [Pang-Yatian/Point-MAE#40]
I pretrain the PointMAE model and then the Acc of the task fintune_scan_hardest can only reach 84.4899.
I would be grateful if you could give me some advice.
please check the attach files that the log I have uploaded
[20230724_105547_pretrain.log](https://github.com/Pang-Yatian/Point-MAE/files/13296558/20230724_105547_pretrain.log)
[20230725_122832_fintune.log](https://github.com/Pang-Yatian/Point-MAE/files/13297296/20230725_122832_fintune.log)

I use the 3090 GPU and here are my virtual environment:
# Name                    Version                   Build  Channel
_libgcc_mutex             0.1                        main  
_openmp_mutex             5.1                       1_gnu  
addict                    2.4.0                    pypi_0    pypi
ansi2html                 1.8.0                    pypi_0    pypi
argparse                  1.4.0                    pypi_0    pypi
attrs                     23.1.0                   pypi_0    pypi
backcall                  0.2.0                    pypi_0    pypi
blas                      1.0                         mkl  
bzip2                     1.0.8                h7f98852_4    conda-forge
ca-certificates           2023.05.30           h06a4308_0  
certifi                   2022.12.7        py37h06a4308_0  
charset-normalizer        3.1.0                    pypi_0    pypi
click                     8.1.3                    pypi_0    pypi
configargparse            1.5.3                    pypi_0    pypi
cudatoolkit               11.3.1              h9edb442_10    conda-forge
cycler                    0.11.0             pyhd3eb1b0_0  
dash                      2.11.0                   pypi_0    pypi
dash-core-components      2.0.0                    pypi_0    pypi
dash-html-components      2.0.0                    pypi_0    pypi
dash-table                5.0.0                    pypi_0    pypi
dbus                      1.13.18              hb2f20db_0  
debugpy                   1.6.7                    pypi_0    pypi
decorator                 5.1.1                    pypi_0    pypi
easydict                  1.10                     pypi_0    pypi
entrypoints               0.4                      pypi_0    pypi
expat                     2.4.9                h6a678d5_0  
fastjsonschema            2.17.1                   pypi_0    pypi
ffmpeg                    4.3                  hf484d3e_0    pytorch
fftw                      3.3.9                h27cfd23_1  
flask                     2.2.5                    pypi_0    pypi
fontconfig                2.14.1               h52c9d5c_1  
freetype                  2.10.4               h0708190_1    conda-forge
glib                      2.69.1               he621ea3_2  
gmp                       6.2.1                h58526e2_0    conda-forge
gnutls                    3.6.13               h85f3911_1    conda-forge
gst-plugins-base          1.14.1               h6a678d5_1  
gstreamer                 1.14.1               h5eee18b_1  
h5py                      3.7.0            py37h737f45e_0  
hdf5                      1.10.6               h3ffc7dd_1  
icu                       58.2                 he6710b0_3  
idna                      3.4                      pypi_0    pypi
importlib-metadata        6.7.0                    pypi_0    pypi
importlib-resources       5.12.0                   pypi_0    pypi
intel-openmp              2021.4.0          h06a4308_3561  
ipykernel                 6.16.2                   pypi_0    pypi
ipython                   7.34.0                   pypi_0    pypi
ipywidgets                8.0.6                    pypi_0    pypi
itsdangerous              2.1.2                    pypi_0    pypi
jedi                      0.18.2                   pypi_0    pypi
jinja2                    3.1.2                    pypi_0    pypi
joblib                    1.2.0                    pypi_0    pypi
jpeg                      9e                   h166bdaf_1    conda-forge
jsonschema                4.17.3                   pypi_0    pypi
jupyter-client            7.4.9                    pypi_0    pypi
jupyter-core              4.12.0                   pypi_0    pypi
jupyterlab-widgets        3.0.7                    pypi_0    pypi
kiwisolver                1.4.4            py37h6a678d5_0  
knn-cuda                  0.2                      pypi_0    pypi
lame                      3.100             h7f98852_1001    conda-forge
ld_impl_linux-64          2.38                 h1181459_1  
libffi                    3.4.4                h6a678d5_0  
libgcc-ng                 11.2.0               h1234567_1  
libgfortran-ng            11.2.0               h00389a5_1  
libgfortran5              11.2.0               h1234567_1  
libgomp                   11.2.0               h1234567_1  
libiconv                  1.17                 h166bdaf_0    conda-forge
libpng                    1.6.37               h21135ba_2    conda-forge
libprotobuf               3.20.3               he621ea3_0  
libstdcxx-ng              11.2.0               h1234567_1  
libtiff                   4.0.9                he85c1e1_2  
libuuid                   1.41.5               h5eee18b_0  
libuv                     1.43.0               h7f98852_0    conda-forge
libxcb                    1.15                 h7f8727e_0  
libxml2                   2.9.14               h74e7548_0  
markupsafe                2.1.3                    pypi_0    pypi
matplotlib                3.2.2                         0  
matplotlib-base           3.2.2            py37hef1b27d_0  
matplotlib-inline         0.1.6                    pypi_0    pypi
mkl                       2021.4.0           h06a4308_640  
mkl-service               2.4.0            py37h402132d_0    conda-forge
mkl_fft                   1.3.1            py37h3e078e5_1    conda-forge
mkl_random                1.2.2            py37h219a48f_0    conda-forge
nbformat                  5.7.0                    pypi_0    pypi
ncurses                   6.4                  h6a678d5_0  
nest-asyncio              1.5.6                    pypi_0    pypi
nettle                    3.6                  he412f7d_0    conda-forge
ninja                     1.11.1                   pypi_0    pypi
numpy                     1.21.5           py37h6c91a56_3  
numpy-base                1.21.5           py37ha15fc14_3  
olefile                   0.46               pyh9f0ad1d_1    conda-forge
open3d                    0.17.0                   pypi_0    pypi
opencv-python             4.7.0.72                 pypi_0    pypi
openh264                  2.1.1                h780b84a_0    conda-forge
openssl                   1.1.1u               h7f8727e_0  
packaging                 23.1                     pypi_0    pypi
pandas                    1.3.5                    pypi_0    pypi
parso                     0.8.3                    pypi_0    pypi
pcre                      8.45                 h295c915_0  
pexpect                   4.8.0                    pypi_0    pypi
pickleshare               0.7.5                    pypi_0    pypi
pillow                    9.5.0                    pypi_0    pypi
pip                       22.3.1           py37h06a4308_0  
pkgutil-resolve-name      1.3.10                   pypi_0    pypi
plotly                    5.15.0                   pypi_0    pypi
pointnet2-ops             3.0.0                    pypi_0    pypi
prompt-toolkit            3.0.38                   pypi_0    pypi
protobuf                  3.20.3           py37h6a678d5_0  
psutil                    5.9.5                    pypi_0    pypi
ptyprocess                0.7.0                    pypi_0    pypi
pygments                  2.15.1                   pypi_0    pypi
pyparsing                 3.0.9            py37h06a4308_0  
pyqt                      5.9.2            py37h05f1152_2  
pyquaternion              0.9.9                    pypi_0    pypi
pyrsistent                0.19.3                   pypi_0    pypi
python                    3.7.16               h7a1cb2a_0  
python-dateutil           2.8.2              pyhd3eb1b0_0  
python_abi                3.7                     2_cp37m    conda-forge
pytorch                   1.10.0          py3.7_cuda11.3_cudnn8.2.0_0    pytorch
pytorch-mutex             1.0                        cuda    pytorch
pytz                      2023.3                   pypi_0    pypi
pyyaml                    6.0              py37h5eee18b_1  
pyzmq                     25.1.0                   pypi_0    pypi
qt                        5.9.7                h5867ecd_1  
readline                  8.2                  h5eee18b_0  
requests                  2.31.0                   pypi_0    pypi
retrying                  1.3.4                    pypi_0    pypi
scikit-learn              1.0.2                    pypi_0    pypi
scipy                     1.7.3            py37h6c91a56_2  
setuptools                65.6.3           py37h06a4308_0  
sip                       4.19.8           py37hf484d3e_0  
six                       1.16.0             pyh6c4a22f_0    conda-forge
sqlite                    3.41.2               h5eee18b_0  
tenacity                  8.2.2                    pypi_0    pypi
tensorboardx              2.2                pyhd3eb1b0_0  
termcolor                 2.1.0            py37h06a4308_0  
threadpoolctl             3.1.0                    pypi_0    pypi
timm                      0.4.5                    pypi_0    pypi
tk                        8.6.12               h1ccaba5_0  
torch-scatter             2.1.1                    pypi_0    pypi
torchaudio                0.10.0               py37_cu113    pytorch
torchvision               0.11.0               py37_cu113    pytorch
tornado                   6.2              py37h5eee18b_0  
tqdm                      4.64.1           py37h06a4308_0  
traitlets                 5.9.0                    pypi_0    pypi
transforms3d              0.4.1                    pypi_0    pypi
typing_extensions         4.6.3              pyha770c72_0    conda-forge
urllib3                   2.0.3                    pypi_0    pypi
wcwidth                   0.2.6                    pypi_0    pypi
werkzeug                  2.2.3                    pypi_0    pypi
wheel                     0.38.4           py37h06a4308_0  
widgetsnbextension        4.0.7                    pypi_0    pypi
xz                        5.4.2                h5eee18b_0  
yaml                      0.2.5                h7b6447c_0  
zipp                      3.15.0                   pypi_0    pypi
zlib                      1.2.13               h5eee18b_0

**Comment by Pang-Yatian** [comment]
Hi,

Randomness is a common issue in point clouds due to the limited size of the dataset. See  [here](https://github.com/Pang-Yatian/Point-MAE/issues/5#issuecomment-1080385674), (https://github.com/CVMI-Lab/PAConv/issues/9) and (https://github.com/ma-xu/pointMLP-pytorch/issues/1).
